common/presenters/single-requests-to-table-component.js

```diff
@@ -7,83 +7,95 @@ const componentService = require('../services/component')
 const moveToCardComponent = require('./move-to-card-component')
 const tablePresenters = require('./table')
 
-function singleRequestsToTable(moves) {
-  const tableConfig = [
-    {
-      head: {
-        text: i18n.t('name'),
-        attributes: {
-          width: '220',
+function singleRequestsToTable({ isSortable = true, query } = {}) {
```

Comment:
I wonder if the default would be cleaner as `false`?

That way the call to these presenters in the middleware feels a little easier to read too. If the table is sortable, you need to pass in the query, if not, call with default options:

```javascript
req.resultsAsTable = {
  active: presenters.singleRequestsToTableComponent({
    isSortable: true,
    query,
  })(cancelledMoves),
  cancelled: presenters.singleRequestsToTableComponent()(cancelledMoves),
}
```